shared/src/test/diff/nu/CaseExpr.mls

```diff
@@ -129,37 +132,40 @@ case then 1 else 0
 :e
 case x, y then x + y
 //│ ╔══[PARSE ERROR] Expected an expression; found a 'then'/'else' clause instead
-//│ ║  l.130: 	case x, y then x + y
+//│ ║  l.133: 	case x, y then x + y
 //│ ╙──       	        ^^^^^^^^^^^^
 //│ ╔══[PARSE ERROR] Expected 'then'/'else' clause after 'case'; found operator application instead
-//│ ║  l.130: 	case x, y then x + y
+//│ ║  l.133: 	case x, y then x + y
 //│ ║         	     ^^^^^^^^^^^^^^^
 //│ ╟── Note: 'case' expression starts here:
-//│ ║  l.130: 	case x, y then x + y
+//│ ║  l.133: 	case x, y then x + y
 //│ ╙──       	^^^^
 //│ ╔══[ERROR] identifier not found: x
-//│ ║  l.130: 	case x, y then x + y
+//│ ║  l.133: 	case x, y then x + y
 //│ ╙──       	     ^
 //│ anything -> ()
 //│ Code generation encountered an error:
 //│   unresolved symbol x
 
 :e
 case (x, y) then x + y
-//│ ╔══[ERROR] Illegal pattern `,`
-//│ ║  l.148: 	case (x, y) then x + y
+//│ ╔══[ERROR] type identifier `,` not found
+//│ ║  l.151: 	case (x, y) then x + y
+//│ ╙──       	      ^^^^
+//│ ╔══[ERROR] type identifier not found: ,
+//│ ║  l.151: 	case (x, y) then x + y
 //│ ╙──       	      ^^^^
-//│ anything -> error
-//│ Code generation encountered an error:
-//│   if expression was not desugared
-
-:e // * FIXME[UCS]
-case [x, y] then x + y
-//│ ╔══[ERROR] type identifier not found: Tuple#2
-//│ ╙──
 //│ nothing -> error
 //│ Code generation encountered an error:
-//│   unknown match case: Tuple#2
+//│   unresolved symbol ,
```

Comment:
Code generator should support `,` operators.

Reply:
Weird, I thought it already did.

Reply:
This is parsed as `if case$scrut is (x, y) then x + y`. So, it's not a problem of new UCS desugarer, we can address problems of parsing and desugaring UCS shorthands in a future.